lib/java/com/google/android/material/slider/res/values/dimens.xml

```diff
@@ -21,6 +21,9 @@
   <dimen name="mtrl_slider_track_side_padding">16dp</dimen>
   <dimen name="mtrl_slider_track_height">4dp</dimen>
 
+  <dimen name="mtrl_slider_tick_radius">1dp</dimen>
+  <dimen name="mtrl_slider_inactive_tick_radius">1dp</dimen>
```

Comment:
Since this is same as the active one, let's remove this and just keep `mtrl_slider_tick_radius` and use it for both activeTickRadius and inactiveTickRadius.

Reply:
Please remove `mtrl_slider_inactive_tick_radius`